app/src/main/java/com/greenfox/opal/gitinder/model/response/Match.java

```diff
@@ -0,0 +1,28 @@
+package com.greenfox.opal.gitinder.model.response;
+
+import java.sql.Timestamp;
+import java.util.ArrayList;
+
+public class Match {
+    String username;
+    Timestamp matched_at;
```

Comment:
@Garlyle `long` or `Date`, check the json converters documentation what the project uses